package.json

```diff
@@ -93,7 +93,6 @@
     "history": "^4.9.0",
     "http-proxy-middleware": "^0.20.0",
     "lodash.camelcase": "^4.3.0",
-    "mousetrap": "^1.6.3",
```

Comment:
We can also remove `@types/mousetrap`, right?

Reply:
yes, will do.